features/apply_with_parameter_store_parameters.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Apply command with parameter_store parameter
+
+  Background:
+    Given a file named "stack_master.yml" with:
+      """
+      stacks:
+        us-east-2:
+          vpc:
+            template: vpc.rb
+      """
+    And a directory named "parameters"
+    And a file named "parameters/vpc.yml" with:
+      """
+      vpc_cidr:
+        parameter_store: CUCUMBER_TEST_VPC_CIDR
+      """
+    And a SSM parameter named "CUCUMBER_TEST_VPC_CIDR" with value "10.0.0.0/16" in region "us-east-2"
```

Comment:
Parameters wouldn't usually be upper case. Maybe change this to something like `/cucumber-test-vpc-cidr` or `/myapp/prod/something-api-key`?

Reply:
Shall do, the uppercase was a hangover where I was actually creating the parameter in SSM parameter store (before I realised that all aws calls are stubbed in test mode) and I didn't want to trip over real parameters.

Reply:
Done: acd7f6f